data/catalogs/artifact_data.yml

```diff
@@ -266,7 +266,7 @@ gebco:
     unit: m+MSL
   path: gebco.tif
 ghs-smod_2015_v2:
-  crs: 54009
+  crs: ESRI:54009
```

Comment:
This is actually issue #696 and I think we should still fix it for main as well. If you need this for tests to pass then we can keep and solve the merge conflicts with main when they come :)

Reply:
They are indeed necessary for these tests. I'll open a separate PR to fix these in main if you want.

Reply:
I think @Tjalling-dejong will or has opened a larger one to update artifact_data catalog (also driver_kwargs should be updated I think)